deepchem/utils/grover.py

```diff
@@ -1,13 +1,94 @@
 import numpy as np
 from typing import List, Any
 from numpy.typing import ArrayLike
-from deepchem.feat.graph_data import BatchGraphData
+from deepchem.feat.graph_data import BatchGraphData, GraphData
 
 try:
     import torch
 except ModuleNotFoundError:
     pass
 
+from deepchem.feat.molecule_featurizers.dmpnn_featurizer import GraphConvConstants
+
+
```

Comment:
Let's add a simple unit test for `BatchGroverGraph` as well

Reply:
added